<launch>
<!-- ############## launch camera ############ -->
	<include file="$(find axis_cam_modified)/launch/axis_basics.launch"/>
	
	<node pkg="tf" type="static_transform_publisher"
        name="axis_camera_init" args="0.6251185950728501 -0.9568288250572693 2.5808307217613855 0.6287990110164235 0.7480819940486101 -0.1563927862398917 -0.14327047964303885) /map /axis_camera_init 30"/>
	
	
<!--   *********************** Module AR_track_alvar *************   -->
	
	<arg name="marker_size" default="12.0" />		# plus gros TODO print  
	<arg name="max_new_marker_error" default="0.08" />
	<arg name="max_track_error" default="0.2" />
	<arg name="cam_image_topic0" default="image_raw" />
	<arg name="cam_info_topic0" default="/camera_info" />		
	<arg name="output_frame0" default="/axis_camera" />

	<node name="ar_track_alvar0" pkg="ar_track_alvar" type="individualMarkersNoKinect" respawn="false" output="screen" args="$(arg marker_size) $(arg max_new_marker_error) $(arg max_track_error) $(arg cam_image_topic0) $(arg cam_info_topic0) $(arg output_frame0)" >
	<remap from="/visualization_marker" to="cam0/visualization_marker"/>
 	</node>
 	
 	
	
<!--   *********************** Server ************* 
 	
 	<arg name="path" default="$(find mark_tracker_tools)/saved_tf" />
	
	<node name="server" pkg="mark_tracker_tools" type="server_tools_mark_tracker.py" respawn="False" args="$(arg path) monica" />
	-->  
	
	
<!--   *********************** rviz *************   
	
	<arg name="rviz_config" default="my_settings.rviz" />
	<arg name="path_rviz" default="$(find mark_tracker_tools)/launch" />
	<node pkg="rviz" type="rviz" name="rviz" args=" $(arg path_rviz)$(arg rviz_config)"/>-->

</launch>
